### PR TITLE
Beets 471 username as email enforced

### PR DESCRIPTION
**Pull request by levertyk**
### Definition of Done:

* [x] Have you checked your [Jira](https://bee-cs458.atlassian.net/jira/software/projects/BEETS/boards/2/backlog) task, to make sure the user acceptance criteria are met?
* [x] Does your code have comments describing its function and reason?

### Source Control

* [x] Is all relevant code checked-in?
* [x] Is this branch ready to be reviewed by the Tech Lead?

### Formatting

* [x] Do your variable names make sense (to others)?
* [x] Have you removed unnecessary Console.logs? 
* [x] Is your code style consistent with other code in the file or the [Style Guide](https://google.github.io/styleguide/jsguide.html)

Screenshot 2023-04-19 at 10 48 03 AM
Screenshot 2023-04-19 at 10 50 09 AM
Screenshot 2023-04-19 at 10 54 32 AM
Screenshot 2023-04-19 at 10 51 16 AM

### CODE REVIEW

**Comment by piratecody** [comment]
Can we change the input field on the create user form (and the login form) to say email instead of username?

**Comment by levertyk** [comment]
> Can we change the input field on the create user form (and the login form) to say email instead of username?

Yep!

Screenshot 2023-04-19 at 1 42 03 PM

**Comment by piratecody** [comment]
image

image

There was no error logged in the server console

**Comment by piratecody** [comment]
image

image

File: client/src/components/Login/Login.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function Login(props) {
         <></>
       )}
       Username:<br></br>
```
Comment (piratecody) [inline client/src/components/Login/Login.js:57]: Could we change the username label to email?

**Review by piratecody — CHANGES_REQUESTED** [review]

**Comment by levertyk** [comment]
> image
> 
> image

Please switch to the new database, a column was added.

**Review by piratecody — APPROVED** [review]

**Pull request merged**